### ISSUE TITLE
`strongly_typed` validation does not work with default parameters

### ISSUE THREAD

**Issue by roo-oliv** [allrod5/parameters-validation#5]
Attempting to use `strongly_typed` validation on default parameters will break the code.

```python
from parameters_validation import validate_parameters, strongly_typed

@validate_parameters
def foo(a: strongly_typed(str) = "default value"):
    print(a)
# Breaks and raises an obscure error
```

**Comment by roo-oliv** [comment]
~A fix for this bug was released in version 1.1.2 and is already available at PyPI.~

**Comment by roo-oliv** [comment]
The issue is still present at `parameters-validation` version 1.1.2.

**Comment by roo-oliv** [comment]
A fix for this bug was released in version 1.1.3 and is already available at PyPI.

**Comment by allrod5** [comment]
Though code no longer breaks when using `strongly_typed` in parameters with default values, no validation is actually performed.
